src/widgets/splits/SplitHeader.cpp

```diff
@@ -170,6 +170,9 @@ namespace {
         if (settings.headerStreamTitle && !s.title.isEmpty())
             title += " - " + s.title;
 
+        // Strip out phantom newlines/trailing whitespace to prevent formatting issues
+        title.replace(QRegExp("[\n\\s]+$"), "");
+
```

Comment:
The 'phantom newlines' can be added through the Twitch website when changing your title by simply pressing enter in the title box.

On the Twitch website, all newlines in the title are stripped.

There's a handy dandy QString function in Qt5 for solving this called `simplified` (see https://doc.qt.io/qt-5/qstring.html#simplified)

```suggestion
        if (settings.headerStreamTitle && !s.title.isEmpty())
        {
            title += " - " + s.title.simplified();
        }

```